services/types.go

```diff
@@ -1,6 +1,7 @@
 package services
 
 type OnboardingTicket struct {
-	ID             string `json:"id"`
-	ExpirationDate int64  `json:"expirationDate,string"`
+	ID                string `json:"id"`
+	ExpirationDate    int64  `json:"expirationDate,string"`
+	StorageQuotaInGiB int    `json:"storageQuotaInGiB,omitempty"`
```

Comment:
Is there a reason why we should not use a `uint` for quota values?

Reply:
There is no such reason i have added condition to validate against negative number so used `int` will remove it and use uint